src/site/content/en/blog/why-coop-coep/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Why you need "cross-origin isolated" for powerful features
+subhead: >
+  Learn why cross-origin isolation is needed to use powerful features such as
+  `SharedArrayBuffer`, `performance.measureMemory()`, and the JS Self-Profiling
+  API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Learn why cross-origin isolation is needed to use
+  powerful features such as `SharedArrayBuffer`, `performance.measureMemory()`,
+  and the JS Self-Profiling API.
+authors:
+  - agektmr
+  - domenic
+hero: hero.jpg
+date: 2020-05-01
+tags:
+  - post
+  - security
+---
+## Introduction
+In [Making your website "cross-origin isolated" using COOP and
+COEP](https://web.dev/coop-coep/) we explained how to adopt to "cross-origin
+isolated" state using COOP and COEP. This is a companion article that explains
+why cross-origin isolation is required to enable powerful features on the browser.
+
+{% Aside 'key-term' %}
```

Comment:
```suggestion
why cross-origin isolation is required to enable powerful features on the browser.

Check out the [Summary](#summary) for a quick explanation of what you need
to do on your site.

{% Aside 'key-term' %}
```

(Optional suggestion that can be ignored but I do feel strongly that it's a good practice)

I really like that summary at the bottom and I think we should make it easier for people to discover. Often times when I read documentation I just want a quick explanation of what I need to do. They'll still need to read the rest of the content to understand how to implement what's described in the summary, but the summary gives them a quick overview of what actions they'll need to take.

Reply:
The longer version of the "summary" is the very article linked at first. Pointing to the "summary" will be a redundant duplicate to that so I will go with current state.